package.json

```diff
@@ -16,11 +16,11 @@
     "pull-stream": "^3.5.0"
   },
   "devDependencies": {
-    "flumecodec": "0.0.0",
-    "flumelog-level": "^0.0.3",
-    "flumelog-memory": "^0.1.2",
-    "flumelog-offset": "^2.0.0",
-    "flumeview-reduce": "^1.2.1",
+    "flumecodec": "0.0.1",
+    "flumelog-level": "^0.0.4",
+    "flumelog-memory": "^0.1.3",
```

Comment:
Side note - we should probably bump all these to version 1, because of weird semver behaviour below 1.
If someone says yes on here I'll go get that done in all the flume modules